From 77582617c9cd7b3ac3bd36bb3bbfeee07d014406 Mon Sep 17 00:00:00 2001
From: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
Date: Tue, 17 Feb 2026 16:59:05 +0100
Subject: [PATCH 1/3] Fix sssd login

Seems like sssd logins fail if they happen after faillock.

Related: https://github.com/flatcar/scripts/pull/3696

Signed-off-by: Krzesimir Nowak <knowak@microsoft.com>
---
 templates/system-auth.tpl | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/templates/system-auth.tpl b/templates/system-auth.tpl
index 905d04f..479bd77 100644
--- a/templates/system-auth.tpl
+++ b/templates/system-auth.tpl
@@ -21,13 +21,11 @@ auth		[success=2 default=ignore]	pam_systemd_home.so
 
 {% if sssd %}
 auth		sufficient	pam_unix.so {{ nullok }} {{ debug }}
+auth		sufficient	pam_sss.so forward_pass {{ debug }}
 {% else %}
 auth		[success=1 new_authtok_reqd=1 ignore=ignore default=bad]	pam_unix.so {{ nullok }} {{ debug }} try_first_pass
 {% endif %}
 auth		[default=die]	pam_faillock.so authfail
-{% if sssd %}
-auth		sufficient	pam_sss.so forward_pass {{ debug }}
-{% endif %}
 {% if caps %}
 auth		optional	pam_cap.so
 {% endif %}
-- 
2.52.0

